utils/data_analysis.py

```diff
@@ -2,11 +2,10 @@
 import logging
```

Comment:
Should we remove this script and get it back in old versions of GDL when ready to reimplement? #138 
Not sure it's relevant to keep it and maintain it for now.

Reply:
Yeah we should remove it

Reply:
okay with you @mpelchat04 and @victorlazio109 ? We can always resuscitate/reimplement it if necessary